fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CompletedSnapshotStoreManager.java

```diff
@@ -96,6 +97,14 @@ public CompletedSnapshotStore getOrCreateCompletedSnapshotStore(TableBucket tabl
                 });
     }
 
+    public void onRemoveCompletedSnapshotStoreByTableBuckets(Set<TableBucket> tableBuckets) {
+        // Remove CompletedSnapshotStore is all we need to do, as remote KV files have already
```

Comment:
I think we can remove theres comments since it may not precise for the root directory of table snapshot  will be drop in coordinator server.